src/components/SearchResultItem.tsx

```diff
@@ -81,46 +81,55 @@ export class SearchResultItem extends React.Component<SearchResultItemProps, Sea
         let ItemContainerStyle = this.state.selected ? "SearchResultItemContainerSelected" : "SearchResultItemContainer";
         let iconPath = this.props.data.iconUrl;
 
-        // The parent of a search result item is the second last entry in 'pathToItem'
-        let parentText = this.props.data.pathToItem[this.props.data.pathToItem.length - 2].text;
```

Comment:
Oh so this is returning null in certain cases?

Reply:
yeah, this.props.data.pathToItem.length was less than 2 in case of sections. So that was being null.

Reply:
yeah... maybe a comment is in order, what is this checking for?

Reply:
Added a comments to that and we have a test for it which was added in the last PR https://github.com/DynamoDS/librarie.js/blob/master/__tests__/mochatest/libraryUtilitiesTest.ts#L119